src/sagas/clients.js

```diff
@@ -1,22 +1,44 @@
 import {all, call, put, select, takeEvery} from 'redux-saga/effects';
-import {createFromProject} from '../clients/gists';
-import {gistExported, gistExportError} from '../actions/clients';
-import {getCurrentProject} from '../selectors';
+import {
+  createGistFromProject,
+  createRepoFromProject,
+} from '../clients/github';
+
+import {
+  gistExported,
+  gistExportError,
+  repoExported,
+  repoExportError,
+  } from '../actions/clients';
+import {getCurrentProject} from '../selectors/';
```

Comment:
Why the trailing `/` here?

Also let’s be consistent about whitespace between `import` statements (at least in the context of the same file)